RealtimeStreamToImpala/README.rst

```diff
@@ -0,0 +1,54 @@
+Realtime Stream To Impala Adapter Configuration
+===============================================
+
+The ETL Realtime Template can be used to create an Adapter that reads from a Realtime Source and persists it to a Sink.
+In this example, we will read events from a Stream in realtime and use a TPFS Sink to make the data queryable by Impala.
+
+The config.json contains a sample Adapter configuration that you can use to accomplish the above task. Our sample Adapter uses the following components:
+
+- etlRealtime Application Template, since we want to perform ETL in realtime
+- Stream source, configured to read from the myStream Stream
+- TPFSAvro sink, configured to write to the converted_stream Dataset
+
+You can create and start the Adapter by using the CDAP CLI (or you can use the UI for a more visual approach)::
+
+  cdap> create adapter trades_conversion RealtimeStreamToImpala/config.json
+  Successfully created adapter 'trades_conversion'
+
+  cdap> start adapter trades_conversion
+  Successfully started adapter 'trades_conversion'
+
+This will create a schedule that will run the Adapter every ten minutes. 
+The next time the Adapter runs, it will spawn a MapReduce job that reads all events added
```

Comment:
@albertshau wrote the original text. @albertshau, do you know what happens here?